tests/parser/types/numbers/test_sqrt.py

```diff
@@ -111,6 +129,33 @@ def test(a: decimal) -> decimal:
 
     c = get_contract(code)
 
-    vyper_sqrt = c.test(sub_val)
-    actual_sqrt = decimal_sqrt(sub_val)
+    vyper_sqrt = c.test(value)
+    actual_sqrt = decimal_sqrt(value)
     assert vyper_sqrt == actual_sqrt
+
+
+class TestFuzzSqrt:
```

Comment:
I think you can drop this class as I *think* hypothesis tests interoperate just fine with pytest fixtures (I may be wrong).

Reply:
I was struggling with hitting the deadline very easily because of redeploying the code every time with get_contract, which is good for our regular tests - but bad for a fuzz test ;)

Reply:
It does interoperate just fine with pytest fixtures, but there are sometimes performance things that I've found to be off. Hypothesis is a hog :pig:

Reply:
Hrm, does hypothesis not respect fixture scopes?  Can you not use a `"module"` scoped fixture so that pytest only runs the deployment once?

Reply:
I wanted to reuse the function scoped get_contract, which blocked me. But I will fix accordingly ( by making a get_contract_mofule).

Reply:
Sorted now :)